network-api/networkapi/wagtailpages/pagemodels/publications/article.py

```diff
@@ -86,6 +88,23 @@ class ArticlePage(FoundationMetadataPageMixin, Page):
         default='image'
     )
 
+    hero_background_color = ColorField(
+        default='#ffffff',
+        help_text='Background color of the hero section.'
+        )
+
+    hero_text_color = models.CharField(
+        max_length=25,
+        choices=[
+            ('light', 'Light'),
+            ('dark', 'Dark'),
+        ],
```

Comment:
**Suggestion:** While this works perfectly fine, it is a quite established pattern to define the choices for a model field as class attributes. Using the class attributes has the advantage that you can refer to the choices in other locations by naming them, rather than by raw value. 

E.g. in a template you could have

```htmldjango
{% if page.hero_text_color == page.HERO_TEXT_COLOR_DARK %}
...
```

 instead of 

```htmldjango
{% if page.hero_text_color == 'dark' %}
...
```

See also: https://docs.djangoproject.com/en/4.0/ref/models/fields/#django.db.models.Field.choices

Reply:
Hi Tibor thanks for another great suggestion, this makes sense to me so I went ahead and implemented it in the PR. 

However, I am not sure if I placed the choices in the correct location or if this is a matter of preference, I was wondering if you could please take a look at how I implemented the choice fields in this file and let me know what you think?

Thanks in advance, I appreciate it!